src/Microsoft.DocAsCode.App/RunServe.cs

```diff
@@ -43,11 +47,104 @@ public static void Exec(string folder, string host, int? port)
             Console.WriteLine($"Serving \"{folder}\" on {url}. Press Ctrl+C to shut down.");
             using var app = builder.Build();
             app.UseFileServer(fileServerOptions);
-            app.Run();
+
+            if (openBrowser)
+            {
+                string relativePath = openBrowserRelativePath;
+                var launchUrl = string.IsNullOrEmpty(relativePath)
+                    ? url
+                    : Path.Combine(url, ResolveOutputHtmlRelativePath(folder, relativePath));
+
+                // Start web server.
+                app.Start();
+
+                // Launch browser process.
+
+                Console.WriteLine($"Launching browser with url: {url}.");
+                using var process = LaunchBrowser(launchUrl);
```

Comment:
When `launchUrl` is an empty string, would this call opens the `url` or an empty browser? Opening `url` seems to be a good fallback to me.

Reply:
In latest commit.
Changed fallback to `url` when fail to resolve relativeUrl.